.travis.yml

```diff
@@ -14,4 +14,4 @@ install:
 
 script:
 - pytest -m "not requires_gpu and not memory_intense and not slow and not travis_slow"
-- python brainscore_language/plugins/test_plugins.py
\ No newline at end of file
+- python brainscore_language/utils/plugins/test_plugins.py
```

Comment:
I think the plugin management is such a central component that I would keep it at the top level